fetch-node.js

```diff
@@ -26,10 +26,14 @@ module.exports = function (context) {
     fetch.Promise = context.Promise;
   }
 
+  if (context && context.useCookie) {
+    fetch = require('fetch-cookie')(fetch);
+  }
+
   return {
     fetch: wrapFetchForNode(fetch),
     Headers: fetch.Headers,
     Request: fetch.Request,
     Response: fetch.Response
   };
-};
+};
```

Comment:
Nitpick: Could you add the newline here? I probably need to make my linter config a little more strict.

Reply:
done.